experimental/segmenter/src/lstm_bies.rs

```diff
@@ -213,9 +217,10 @@ impl<'l> Lstm<'l> {
     /// `word_segmenter` is a function that gets a "clean" unsegmented string as its input and returns a BIES (B: Beginning, I: Inside, E: End,
     /// S: Single) sequence for grapheme clusters. The boundaries of words can be found easily using this BIES sequence.
     pub fn word_segmenter(&self, input: &str) -> Box<[Bies]> {
-        // input_seq is a sequence of id numbers that represents grapheme clusters or code points in the input line. These ids are used later
-        // in the embedding layer of the model.
-        // Already checked that the name of the model is either "codepoints" or "graphclsut"
+        #![allow(clippy::unwrap_used)] // unwraps justified
```

Comment:
Nit: Avoid a blanket suppression; it should go on each line with a comment. You already have comments like `// embedding has shape (dict.len() + 1, hunit), g_id is at most dict.len()` so just make them associated with line-by-line suppressions